pha-admin/src/main/java/com/store/pharmacy/Client/model/Client.java

```diff
@@ -1,69 +1,61 @@
 package com.store.pharmacy.Client.model;
```

Comment:
Rename this package name to match the regular expression ^[a-z_]+(.[a-z_][a-z0-9_])$